src/components/DataTable/DataTable.jsx

```diff
@@ -16,6 +17,48 @@ const {
 
 const cx = lucidClassNames.bind('&-DataTable');
 
+//SUBCOMPONENT - HatchPattern
+const HatchPattern = createClass({
```

Comment:
I think it belongs in its own file too, just not sure where to put it; should it live alongside DataTable?